### ISSUE TITLE
multi-combobox (with Angular) - Event `selection-change` is not emitted -> error indicated by UI5 in my IDE

### ISSUE THREAD

**Issue by Abneco** [SAP/ui5-webcomponents#5341]
Hello,

I work with Angular 14 and I want to use the multi-combobox component, the following code works. But my IDE (Intellij) tells me this:
Event `selection-change` is not emitted by any applicable directives nor by `ui5-multi-combobox` element.
note: the event takes place, and I can recover the elements.

I went to see the documentation of the component
https://sap.github.io/ui5-webcomponents/playground/components/MultiComboBox/

I tried the following items
`(selectionchange)` does not show an error but does not work
`(selectionChange)` says there is an error but works.
`(selection-change)` says there is an error but works.
`(change)` I don't show an error but it doesn't work

Can you tell me if I am doing something wrong or if there is another way to call the event without throwing an exception? thank you in advance for your return.

````typescript

        <!-- sélecteur de catégorie d'article -->
        <div class="form-section">
            <ui5-label for="articleTypeInput" required show-colon>Catégorie de l'article</ui5-label>
            <ui5-multi-combobox #articleTypesInput id="articleTypesInput" name="articleTypes" placeholder="Sélectionner la catégorie de l'article" [attr.readonly]="" [attr.value-state]="status" (selection-change)="selectType($event)" >
                <ui5-mcb-item data-index="ZERO" text=">_ SELECT EMPTY"></ui5-mcb-item>
                <ui5-mcb-item data-index="ALL" text=">_ SELECT ALL"></ui5-mcb-item>
                <ui5-mcb-item *ngFor="let type of types ; let i = index" [attr.data-index]="i+1" [attr.text]="type" [attr.selected]="hasType(type)" [attr.additional-text]=""></ui5-mcb-item>
                <div *ngIf="status=='Error'" slot="valueStateMessage">Sélectionner au moins une catégorie.</div>
            </ui5-multi-combobox>
        </div>

````

**Comment by MapTo0** [comment]
Hello,

Is this happening for the `ui5-multi-combobox` only? Is the issue available for other components as well?

Regards,
Martin

**Comment by Abneco** [comment]
I had no problem with the simple combobox, only with the mutli combobx component


![image](https://user-images.githubusercontent.com/29584029/172851645-df319141-db90-4164-9272-7996a8389248.png)

**Comment by MapTo0** [comment]
What happens if you try to use the `selection-change` event to the combobox? 
I see lots of issue reported for the Intellij when working with web components.
It somehow wants to know all the events supported by the custom element but we do not provide definitions for them.

https://stackoverflow.com/questions/60702492/how-to-permanently-add-an-event-like-touchstart-to-all-elements-in-webstorm-in
https://youtrack.jetbrains.com/issue/WEB-44387

**Comment by Abneco** [comment]
1. What happens if you try to use the event to the combobox?
I simply retrieve the list of selected categories
![image](https://user-images.githubusercontent.com/29584029/172880683-7495a9df-096e-4522-926f-a23d441c5988.png)


2. the tag you indicated on the stackoverflow link seems to work, this hides the problem, this will suffice for now...
````
        <!-- sélecteur de catégorie d'article -->
        <div class="form-section">
            <ui5-label for="articleTypeInput" required show-colon>Catégorie de l'article</ui5-label>
            <!--suppress AngularUndefinedBinding -->
            <ui5-multi-combobox #articleTypesInput id="articleTypesInput" name="articleTypes" placeholder="Sélectionner la catégorie de l'article" [attr.readonly]="" [attr.value-state]="statusTypeArticle" (selectionChange)="selectType($event)">
                <ui5-mcb-item data-index="ZERO" text=">_ SELECT EMPTY"></ui5-mcb-item>
                <ui5-mcb-item data-index="ALL" text=">_ SELECT ALL"></ui5-mcb-item>
                <ui5-mcb-item *ngFor="let type of types ; let i = index" [attr.data-index]="i+1" [attr.text]="type" [attr.selected]="hasType(type)" [attr.additional-text]=""></ui5-mcb-item>
                <div *ngIf="statusTypeArticle=='Error'" slot="valueStateMessage">{{messageError}}</div>
            </ui5-multi-combobox>
        </div>
````

I just had a strange bug... the following `(selection-change)` event didn't work anymore, so I switched to the other way `(selectionChange)` of noting... strange

anyway I have no more errors, it's ok for me

**Comment by MapTo0** [comment]
We do not suppress any of the events and it should work with `selection-change` as well.
Good that it works for you, I am closing the issue as the problem is related more to the IDE than the `ui5-webcomponents` itself.

Regards,
Martin

**Comment by Abneco** [comment]
yes, I suspected that it was a problem on my side, but I did not understand why, I remembered to have made a test one evening, I removed this from my code, and `selection-change` it worked again.

```typescript
<script data-ui5-config type="application/json">
    {
        "theme": "sap_fiori_3",
        "noConflict": {
            "events": ["selection-change", "header-click"]
        }
    }
</script>
```

I always use this with `<!--suppress AngularUndefinedBinding -->`, but at least my code is compliant with the documentation.
Thanks for your help, it avoids constant errors in my IDE, and I appreciate that.
